### ISSUE TITLE
loss for xbd dataset 

### ISSUE THREAD

**Issue by oshholail** [nka77/DamageAssessment#3]
which loss is used in training xbd dataset ? i try all focal loss as mentioned 
but get this error :
 in soft_dice_score
    assert output.size() == target.size()
AssertionError

**Comment by nka77** [comment]
Any of the losses should work fine. In order to use the defined loss functions, the model output image and the target image are expected to be of same size.
